include/Operator/SecondQuantized/USQHamiltonian.hpp

```diff
@@ -0,0 +1,250 @@
+// This file is part of GQCG-gqcp.
+// 
+// Copyright (C) 2017-2019  the GQCG developers
+// 
+// GQCG-gqcp is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// GQCG-gqcp is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-gqcp.  If not, see <http://www.gnu.org/licenses/>.
+// 
+#pragma once
+
+
+#include "Operator/SecondQuantized/SQHamiltonian.hpp"
+
+
+namespace GQCP {
+
+
+/**
+ *  A class for representing the unrestricted second-quantized electronic Hamiltonian
+ *  It consists of two restricted Hamiltonians for both the beta and alpha component and a mixed two-electron term
```

Comment:
```suggestion
```
I do not think it is needed to provide comments about internals in the class description: this can be put in the documentation manual on the website.